scripts/Muon_Analysis_2.py

```diff
@@ -41,12 +43,38 @@ def qapp():
     return _app
 
 
-app = qapp()
-try:
-    ex = MuonAnalysis2Gui()
-    ex.resize(700, 700)
-    ex.show()
-    app.exec_()
-except RuntimeError as error:
-    ex = QtGui.QWidget()
-    QtGui.QMessageBox.warning(ex, "Muon Analysis version 2", str(error))
+def main():
+    app = qapp()
+    try:
+        muonGUI = MuonAnalysis2Gui()
+        muonGUI.resize(700, 700)
+        muonGUI.show()
+        app.exec_()
+        return muonGUI
+    except RuntimeError as error:
+        muonGUI = QtGui.QWidget()
+        QtGui.QMessageBox.warning(muonGUI, "Muon Analysis version 2", str(error))
+        return muonGUI
+
+
+def saveToProject():
+    if muonGUI is None:
+        return ""
+    project = "test"
+    return project
+
+
+def loadFromProject(project):
+    muonGUI = main()
+    muonGUI.dockWidget.loadFromProject(project)
+    return muonGUI
+
+if __name__ == '__main__':
+    muonGUI = main()
+    #test 2
```

Comment:
Can we remove the commented-out code?

Reply:
sure it was just there to make testing easier.

Reply:
Sure. We just want to make sure it doesn't end up in the `master`.